Weather/FlagoftheCharles.15m.rb

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/ruby
+
+# <bitbar.title>Flag of the Charles</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Isaac</bitbar.author>
+# <bitbar.author.github>irstacks</bitbar.author.github>
+# <bitbar.desc>Shows Boston's current Community Boating, Inc flag and related weather information.</bitbar.desc>
+# <bitbar.image>http://imgur.com/gQ3suo0</bitbar.image>
```

Comment:
Can you please change this to point to the actual PNG, so `https://i.imgur.com/gQ3suo0.png`